specification/dimensions/resourcetype.md

```diff
@@ -0,0 +1,38 @@
+# Resource Type
+
+Resource Type describes the kind of resource for which you are being charged.  A Resource Type is commonly used for scenarios like identifying cost changes in groups of similar resources.
```

Comment:
@cnharris10 Is the intent of this to be a human-readable display name (e.g., "Virtual machines") or to be the identifier the provider uses to indicate the resource's type (e.g., "Microsoft.Compute/virtualMachines")? If we don't care, this is fine as is. If you want to force it to be human-readable, we should call that out explicitly. We should also determine if we want to allow for the ID (even if after 1.0), given that's what providers (at least Microsoft) make available today.

Reply:
Yeah, I think we have to provide some guidance for the value providers should use given that no one seems to have exactly what we're proposing in supporting content.

Reply:
It's meant to be open-ended and up to the provider to define. Human-readable or coded is fine. I will add 2 points around:

- up to the provider to create values
- value SHOULD be consistent in one form (ex: human-readable or coded) but not mixed.

Reply:
Should we ask practitioners what they want here?